app/lib/actions/katello/content_view/publish.rb

```diff
@@ -4,6 +4,7 @@ module ContentView
       class Publish < Actions::EntryAction
         middleware.use Actions::Middleware::KeepCurrentUser
 
+        # rubocop:disable MethodLength
```

Comment:
Why are you disabling this cop here? If we think that the methods of dynflow actions should not be checked, maybe we should disable it in app/lib/actions.